src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,57 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const clickHandler = (tab: Tab) => {
```

Comment:
useCallback

Reply:
@ihor-jpeg, As I understand, `useCallback ` should be used for a method which is passed as a props to another component (so that the child component does not re-render each time a parent component is rendered and a new copy of method is created).

In the current scenario, the `clickHandler ` method is not being passed to any other component, so does it still add any value to use `useCallback`

Reply:
yes, because the we re-render the component we don't want to create a new function in the memory every time

Reply:
Done.